app-embed-example/docker-compose.yml

```
@@ -1,25 +1,45 @@
-version: "3"
+version: '3.9'
 services:
   metabase:
-    image: metabase/metabase-enterprise:latest
-    environment:
-      - MB_PREMIUM_EMBEDDING_TOKEN=${MB_PREMIUM_EMBEDDING_TOKEN}
-      - MB_ENABLE_EMBEDDING=true
-      - MB_EMBEDDING_APP_ORIGIN=http://${HOST}:3001
-      - MB_JWT_ENABLED=true
-      - MB_JWT_IDENTITY_PROVIDER_URI=http://${HOST}:3001/api/auth/metabase
-      - MB_JWT_SHARED_SECRET=ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff
-      - MB_SAML_ENABLED=${MB_SAML_ENABLED}
-      - MB_SAML_APPLICATION_NAME=${MB_SAML_APPLICATION_NAME}
-      - MB_SAML_IDENTITY_PROVIDER_URI=${MB_SAML_IDENTITY_PROVIDER_URI}
-      - MB_SAML_IDENTITY_PROVIDER_CERTIFICATE=${MB_SAML_IDENTITY_PROVIDER_CERTIFICATE}
+    image: metabase/metabase-enterprise:v1.43.2
+    container_name: metabase_full_app_embedding_demo
+    hostname: metabase
+    volumes: 
+    - /dev/urandom:/dev/random:ro
     ports:
-      - "3000:3000"
-  metalbase:
+      - 3000:3000
+    environment: 
+      - "MB_SITE_URL=http://localhost:3000"
+      # - "MB_SESSION_COOKIE_SAMESITE=None"
+      - "MB_EMBEDDING_APP_ORIGIN=*"
+      - "MB_ENABLE_EMBEDDING=true"
+      - "MB_PREMIUM_EMBEDDING_TOKEN=XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+      - "MB_JWT_ENABLED=true"
+      - "MB_JWT_IDENTITY_PROVIDER_URI=http://localhost:8080/api/auth/metabase"
+      - "MB_JWT_SHARED_SECRET=ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff"
+    networks: 
+      - metanet1
+  setup_full_app_embedding_demo:
+    image: bash:5.1.16
+    container_name: setup_full_app_embedding_demo
+    volumes:
+      - $PWD/setup:/tmp
+    networks:
+      - metanet1
+    depends_on:
+      - metabase_full_app_embedding_demo
+    command: sh /tmp/metabase-setup.sh metabase:3000
```

Since it's actually the Site URL, then explicitly use that.
```suggestion
    command: sh /tmp/metabase-setup.sh http://localhost:3000
```

---

this is a setup container that runs in an internal network, so you need to point to the metabase container that lives on that network rather than localhost